index.js

```diff
@@ -2,7 +2,7 @@ const core = require('@actions/core');
 const child_process = require('child_process');
 const fs = require('fs');
 const crypto = require('crypto');
-const { home, sshAgent, sshAdd } = require('./paths.js');
+const { home, sshAgent, sshAdd, gitPath } = require('./paths.js');
```

Comment:
Thanks for fast review @sebastiankugler 🤗 

If consistency is paramount, if you allow, I would rather go with more precise naming for others so `homePath`, `sshAgentPath` and `sshAddPath` as that would be significantly better naming imo

Cheers!

Reply:
No general objections, and I think it is a useful suggestion :+1:. I have no clear preference regarding the `Path` suffix, but tend to follow the suggestion and use `Path` for all the other variables as well. It would be great @ochococo if you could update this.